src/utils/template.data.ts

```diff
@@ -0,0 +1,11 @@
+import { isActiveFeature } from "./feature.flag";
+import * as config from "../config";
+import { getUrlWithParamsToPath } from "./url";
+import { Request } from "express";
+
+export const addActiveSubmissionBasePathToTemplateData = (templateData: Object, req: Request) => {
+  if (isActiveFeature(config.FEATURE_FLAG_ENABLE_REDIS_REMOVAL)) {
+    templateData["FEATURE_FLAG_ENABLE_REDIS_REMOVAL"] = true;
```

Comment:
I preferred the old way of setting the field data - less verbose, without []s and ""s. Is there an advantage to doing it this way?

Reply:
not sure, it might be safer with undefined's but i can ind out

Reply:
👍 if we can avoid the extra characters would be good.

Reply:
can't use the dot as it says property undefined on type Object, so the type has to have the field names in it. Not sure why it seemed to be ok before though.

Reply:
Oh well.